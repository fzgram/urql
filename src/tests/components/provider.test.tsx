/* tslint:disable */

import React from 'react';
import Provider from '../../components/provider';
import renderer from 'react-test-renderer';

describe('Provider Component', () => {
  it('should provide a context consumer and pass through to client', () => {
    // @ts-ignore
    const component = renderer.create(
      // @ts-ignore
      <Provider client="test">
        <div />
      </Provider>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
