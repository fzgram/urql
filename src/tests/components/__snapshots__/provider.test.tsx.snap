// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Provider Component should provide a context consumer and pass through to client 1`] = `<div />`;
